### ISSUE TITLE
news.php  -  problem with e_PAGETITLE

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#1472]
On the news.php (default, just list of all news) e_PAGETITLE is set to the name of the first news category.  
Action and template is 'list' , if this helps.

Shoudln't be this always start with PAGE_NAME  (News) and in category list just add category name and in news detail add news title?

**Comment by Jimmi08** [comment]
@Moc Could you label this as bug, this is not correct behaviour. Thanks

**Comment by Jimmi08** [comment]
@CaMer0n Thanks. Tested. It works expert news.php (action is empty)
I think that problem is in this lines:
```
switch($action)
{
	case 'item':
	//	setNewsFrontMeta($newsAr[1]);
		break;
	
	case 'list':
	default:
  	setNewsFrontMeta($newsAr[1], 'category');
	break;
}
```
setNewsFrontMeta() has no solution for 'category'. 
If I deleted calling this function for default (it is not used for list), it works as should it has. 

So:
action= list  route= news/list/category  correct
action= extend  route= news/list/items  correct
action= month  route= news/list/month  -  without title, maybe it should be this way
action=    route= news/list/items  - wrong, solution from above helped
